src/styles/blocks/card.scss

```
@@ -0,0 +1,98 @@
+.card {
+  width: 200px;
+  padding: 32px 16px 17px;
+  border-radius: $border-radius-default;
+  border: 1px solid $color-white-smoke;
+
+  &__image {
+        width: 160px;
+        height: 134px;
+  }
+
+  &__link {
+    margin-bottom: 40px;
+    display: flex;
+    width: 100%;
+    align-items: center;
+    justify-content: center;
+  }
+
+  &__name {
+    display: block;
+    color: $color-midnight-express;
+    font-size: 12px;
+    font-weight: 500;
+    text-decoration: none;
+    margin-bottom: 4px;
+    line-height: 18px;
+  }
+
+  &__code {
+    color: $color-comet;
+    font-size: 10px;
+    font-weight: 400;
+    margin-top: 0;
+    margin-bottom: 16px;
```

why do you need to reset `margin-top`?
```suggestion
    margin-bottom: 16px;
```